### ISSUE TITLE
Spellcheck not working proberly in 0.6.0

### ISSUE THREAD

**Issue by melltuga** [TheGoddessInari/hamsket#203]
As far as I can see, so far there's no way of changing the spellchecker language or turning it off, which means, that for me (writing in four different languages) basically everything is underlined in red all the time.

Or is it just me?

thanks!<3

**Comment by TheGoddessInari** [comment]
Right, I'll try to find time to work on this. It's just the basic electron enabled spellcheck.

> By default the spellchecker will enable the language matching the current OS locale.

The general problem is that the language resource codes don't usually match 1:1 with the spellcheck language codes, and it's all per-session rather than per-window, and ExtJS always makes everything more complicated, so I'll have to wire something up.
